### PR TITLE
reformatted arg_names #2410

### PR DESCRIPTION
**Pull request by Youssef-Saleh**
Reformatted the function arg_names #2410 from Reformat general #791, let me know if there's anything else I can do.

### CODE REVIEW

File: ivy/functional/ivy/general.py [hunk 1]
```diff
@@ -1679,13 +1679,31 @@ def try_else_none(fn):
...
     Parameters
     ----------
     receiver
+        Function or class constructor
+    
+    Returns
+    -------
+    ret
+        List containing the keyword arguments' names for a function or class constructor
+
```
Comment (Aarsh2001) [inline ivy/functional/ivy/general.py:1694]: please use linter to add in appropriate spaces here

File: ivy/functional/ivy/general.py [hunk 2]
```diff
@@ -1679,13 +1679,31 @@ def try_else_none(fn):
...
+    Examples
+    --------
+    >>> x = ivy.arg_names(ivy.tan)
+    >>> print(x)
+    ['x', 'out']
+
+    >>> x = ivy.arg_names(ivy.optimizers.Adam)
+    >>> print(x)
+    ['lr', 'beta1', 'beta2', 'epsilon', 'inplace', 'stop_gradients', 'compile_on_next_step', 'device']
```
Comment (Aarsh2001) [inline ivy/functional/ivy/general.py:1703]: Here as well

**Review by Aarsh2001 — CHANGES_REQUESTED** [review]
Hi @Youssef-Saleh, thank you for the contribution. Just a minor change, else the PR is good to go. Thanks🙂

File: ivy/functional/ivy/general.py (hunk 2 above)
Comment (Youssef-Saleh) [reply ivy/functional/ivy/general.py]: Thank you for the suggestions, I used pylint and followed the linter's suggestions, let me know if there's anything else

**Comment by Aarsh2001** [comment]
lgtm!

**Pull request merged**